apps/api/src/modules/applications/applications.resolver.ts

```diff
@@ -0,0 +1,15 @@
+import { Resolver, Query, Args } from '@nestjs/graphql';
+import { ApplicationsService } from './applications.service';
+import { Application } from './entities/application.entity';
+
+@Resolver(() => Application)
+export class ApplicationsResolver {
```

Comment:
We shouldn't have an unprotected resolver for this. I also think we do not need this for the time being. If possible, we can remove this resolver entirely for now (along with the spec file) and add it later when needed.

Reply:
Removed this